.circleci/config.yml

```diff
@@ -61,6 +61,16 @@ jobs:
       - run-bazel-rbe:
           command: bazel test $(bazel query 'kind(checkstyle_test, //...)')
 
+  build-unused-deps:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run:
+          command: bazel run @graknlabs_build_tools//unused_deps -- list
+          no_output_timeout: 30m
```

Comment:
Can we make this run statement as part of the `build` job, @vmax ?

Reply:
@haikalpribadi fixed in https://github.com/graknlabs/client-java/pull/56/commits/300d686ceade3d837b82bcab63e08a001c3a3e18